virtual-server/examples/python/README.md

```diff
@@ -0,0 +1,28 @@
+# Python example
+
+Python example for talking to a VirtualServer custom controller and kubevirt.
+
+The example does:
+1. Remove existing virtual server.
+2. Create a new virtual server based on `my_virtualserver` configuration and wait until it's ready. The instance is started automatically.
+3. Stop the instance and wait until is fully stopped.
+4. Start the instance and wait until is ready.
+5. Delete the virtual server.
```

Comment:
The python example illustrates the following:
1. Removal of an existing Virtual Server.
2. Creation of a new Virtual Server based on the `my_virtualserver` example configuration
3. Waiting for a Virtual Server ready status.
3. Stop the Virtual Server instance and wait until it is stopped.
4. Start the Virtual Server instance and wait until is ready.
5. Delete the Virtual Server instance.